docker-build/src/test/java/io/quarkus/ts/docker/DockerBuildIT.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.ts.docker;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.dockerjava.api.model.Image;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.utils.DockerUtils;
+
+@QuarkusScenario
+@DisabledOnNative(reason = "Due to reduce native execution time")
```

Comment:
`NativeConfigurationIT` is executed in native, the native binary gets produced in native run of the TS, probably no harm to have the test enabled.